src/app/components/Application/Application.jsx

```diff
@@ -92,44 +69,45 @@ class Application extends React.Component {
     );
 
     return (
-      <DocumentTitle title="Shared Collection Catalog | NYPL">
-        <div className="app-wrapper">
-          <Header
-            navData={navConfig.current}
-            skipNav={{ target: 'mainContent' }}
-            patron={this.state.patron}
-          />
-          <DataLoader
-            location={this.context.router.location}
-            next={Store.next}
-            key={JSON.stringify(dataLocation)}
-          >
-            {React.cloneElement(this.props.children, this.state.data)}
-          </DataLoader>
-          <Footer />
-          <Feedback submit={this.submitFeedback} />
-        </div>
-      </DocumentTitle>
+      <MediaContext.Provider value={this.state.media}>
+        <DocumentTitle title="Shared Collection Catalog | NYPL">
+          <div className="app-wrapper">
+            <Header
+              navData={navConfig.current}
+              patron={this.state.patron}
```

Comment:
Should this really be `this.state.patron` and not `this.props.patron`. If the patron updates in the store I think this will not update, right?